### ISSUE TITLE
getFromJson error on returned value

### ISSUE THREAD

**Issue by remi-dandois** [cerberustesting/cerberus-source#2354]
Hello
I am facing a new behaviour with the "getFromJson" action.
1. I call an API
2. I use the "getFromJson" action to get an object from the API response
3. Cerberus find the object from the API response
4. The result is saved but in a wrong format
5. I can not use the object for my assessements

### API response : 
```json
{
  "sellsWoodBasedProducts": false,
  "users": [{
    "profile": "",
    "lastName": "Dedicated",
    "firstName": "Seller",
    "email": "test@ext.adeo.com",
    "preferredLang": "FR"
  }],
  "company": {
    "ecoPartRegistrations": {
      "battery": "25",
      "packaging": "14",
      "electronicProducts": "36",
      "furniture": "47",
      "textileProduct": "58",
      "waste": "69"
    }
  }
}
```

### Expected value:
`{"battery": 25, "packaging": 14, "electronicProducts": 36, "furniture": 47, "textileProduct": 58, "waste": 69}`
### Calculated value by Cerberus:
`{battery=25, packaging=14, electronicProducts=36, furniture=47, textileProduct=58, waste=69}'`

**Comment by vertigo17** [comment]
getFromJson just like getFromXml generate a user readable string in order to make controls easier.
If we want to retrieve a json format, we should create getRawFromJson just like we created getRawFromXml.
That way, using we would still encourage users to keep it simple not parsing several times XML or JSON structures but still making it possible to do it with Raw functions in case this is mandatory.

**Comment by vertigo17** [comment]
Will be fixed with #2355
